conf.py

```diff
@@ -351,8 +351,11 @@ def xapian_run_example_command(ex):
         return "%s -unsafe -target:exe -out:%s.exe %s -r:XapianSharp.dll\n./%s.exe" \
             % (csc, ex, xapian_code_example_filename(ex), ex)
     elif highlight_language == 'java':
-        javac = get_tool_name('JAVAC', 'javac')
-        java = get_tool_name('JAVA', 'java')
+        java_bindings_dir = get_tool_name('JAVA_BINDINGS_DIR', 'usr/local/') # change default
+        classpath = ' -classpath ' + java_bindings_dir + 'xapian_jni.jar:./code/java/'
+        java_library_path = ' -Djava.library.path=' + java_bindings_dir + '../.libs'
+        javac = 'javac' + classpath
+        java = 'java' + java_library_path + classpath
```

Comment:
But these still need to use `get_tool_name`. For instance, someone might want to use a different compiler entirely.